extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -90,7 +90,7 @@ public class JibProcessor {
     private static final IsClassPredicate IS_CLASS_PREDICATE = new IsClassPredicate();
     private static final String BINARY_NAME_IN_CONTAINER = "application";
 
-    private static final String JAVA_17_BASE_IMAGE = "registry.access.redhat.com/ubi8/openjdk-17-runtime:1.17";
+    private static final String JAVA_17_BASE_IMAGE = "registry.access.redhat.com/ubi8/openjdk-17-runtime:1.16";
     private static final String JAVA_11_BASE_IMAGE = "registry.access.redhat.com/ubi8/openjdk-11-runtime:1.16";
```

Comment:
We had a few complaints about that. Note that these images are configurable. We can switch to "latest" and inform the user of the change (it would be a breaking change). Initially, we were using the complete version (but that required too much work).

Note that, I had (but it seems to be broken) automated tests when I updated. Basically: build a container with jib and run it, curl and exit.